package.json

```diff
@@ -11,10 +11,10 @@
     "url": "akameco.github.io"
   },
   "scripts": {
-    "add-contributor": "add-contributor add",
+    "add-contributor": "all-contributors add",
     "precommit": "yarn flow && lint-staged",
     "build": "babel src -d lib --ignore __tests__,__fixtures__",
-    "prepare": "yarn build",
+    "prepack": "yarn build",
```

Comment:
Why?

Reply:
Because this way you support intermediate states when you are working on a PR and are testing the lib in another App, so you are depending on it using a git dependency and quoting [npm scripts docs](https://docs.npmjs.com/misc/scripts):
> prepack: run BEFORE a tarball is packed (on npm pack, npm publish, and **when installing git dependencies**)